linux/open_vm_tools/ovt_verify_pkg_install.yml

```diff
@@ -0,0 +1,84 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This test case is to verify open-vm-tools installation with guest OS package manager.
+# If guest OS doesn't have open-vm-tools, this case will install open-vm-tools from OS ISO image
+# (e.g. RHEL, SLES, SLED, etc),or install from official online repository (e.g. Ubuntu, Debian,
+# VMware Photon OS, CentOS, etc).
+# If it already has open-vm-tools installed from package, and update_tools parameter is set "true"
+# in vars/test.yml, it will reinstall open-vm-tools.
+# If it already has open-vm-tools not installed from package, this test case will be skipped with
```

Comment:
This description "If it already has open-vm-tools not installed from package" seems need to be refined.

Reply:
Updated.